EMULATORS.md

```diff
@@ -53,3 +53,4 @@
 | [Kotlin-Gameboy-Emulator](https://github.com/stan-roelofs/Kotlin-Gameboy-Emulator) | Kotlin | Cross platform DMG emulator |
 | [Emux](https://github.com/Washi1337/Emux) | C# | |
 | [Y2Dmg](https://github.com/Y2JB/DMG) | C# | Fully featured DMG emulator written in C#. Passes Blargg's test roms and supports most games. Core is cross platform (.net core), renderer is Windows only.|
+| [Gandalf](https://github.com/stan-roelofs/Gandalf) | C++ | GB, GBC emulator with an SDL/ImGui frontend. |
```

Comment:
Thanks for the add! Maybe you can mention what makes the emulator unique? Like the debugging tools?